tools/RAiDER/models/hrrr.py

```diff
@@ -345,6 +367,26 @@ def __init__(self):
         # The projection information gets read directly from the  weather model file but we
         # keep this here for object instantiation.
         self._proj = HRRR_AK_PROJ
+        self.setLevelType('ml')
+
+
+    def setLevelType(self, levelType):
```

Comment:
With this function defined in the main weatherModel module, we don't need it defined twice. Then you would just overload the `self.__load_pressure_levels()` method in hrrr.py to raise the same `NotImplementedError` you have here.